tests/clone3-report-ns-id.c

```diff
@@ -0,0 +1,42 @@
+/*
+ * Check appending auxstr of clone3(2) when --namespace=switchTo is given
+ *
+ * Copyright (c) 2024 The strace developers.
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ */
+
+#include "tests.h"
+#include "xmalloc.h"
+
+#include <limits.h>
+#include <linux/sched.h>
+#include <stdio.h>
+#include <sys/syscall.h>
```

Comment:
sys/syscall.h is provided by libc, and we don't use it.

Reply:
fixed locally.